projects/02-data-lovers/src/data/got/got.json

```
@@ -0,0 +1,551 @@
+        "firstName": "Cateyln",
+        "lastName": "Stark",
+        "fullName": "Catelyn Stark",
+        "title": "Lady of Winterfell",
+        "family": "House Stark",
+        "imageUrl": "https://thronesapi.com/assets/images/catelyn-stark.jpg",
+        "born": "264 DC",
+        "death": "299 DC"
+      },
+      {
+        "id": 11,
+        "firstName": "Robb",
+        "lastName": "Stark",
+        "fullName": "Rob Stark",
+        "title": "Lord of Winterfell",
+        "family": "House Stark",
+        "imageUrl": "https://thronesapi.com/assets/images/robb-stark.jpg",
+        "born": "283 DC",
+        "death": "299 DC"
+      },
+      {
+        "id": 12,
+        "firstName": "Theon",
+        "lastName": "Greyjoy",
+        "fullName": "Theon Greyjoy",
+        "title": "Captain of Sea Bitch",
+        "family": "House Greyjoy",
+        "imageUrl": "https://thronesapi.com/assets/images/theon.jpg",
+        "born": "278 DC"
+      },
+      {
+        "id": 13,
+        "firstName": "Joffrey",
+        "lastName": "Baratheon",
+        "fullName": "Joffrey Baratheon",
+        "title": "Lord of the Seven Kingdoms, Protector of the Realm",
+        "family": "House Lanister",
+        "imageUrl": "https://thronesapi.com/assets/images/joffrey.jpg",
+        "born": "286 DC",
+        "death": "300 DC"
+      },
+      {
+        "id": 14,
+        "firstName": "Tyrion",
+        "lastName": "Lannister",
+        "fullName": "Tyrion Lannister",
+        "title": "Hand of the Queen",
+        "family": "House Lanister",
+        "imageUrl": "https://thronesapi.com/assets/images/tyrion-lannister.jpg",
+        "born": "272 DC"
+      },
+      {
+        "id": 15,
+        "firstName": "Sandor",
+        "lastName": "Clegane",
+        "fullName": "The Hound",
+        "title": "The Hound",
+        "family": "House Clegane",
+        "imageUrl": "https://thronesapi.com/assets/images/the-hound.jpg",
+        "born": "270 DC",
+        "death": "300 DC"
+      },
+      {
+        "id": 16,
+        "firstName": "Petyr",
+        "lastName": "Baelish",
+        "fullName": "Petyr Baelish",
+        "title": "Littlefinger",
+        "family": "House Baelish",
+        "imageUrl": "https://thronesapi.com/assets/images/littlefinger.jpg",
+        "born": "264 AL",
+        "death": "305 AL"
+      },
+      {
+        "id": 17,
+        "firstName": "Davos",
+        "lastName": "Seaworth",
+        "fullName": "Davos Seaworth",
+        "title": "Hand of the King",
+        "family": "House Seaworth",
+        "imageUrl": "https://thronesapi.com/assets/images/davos-seaworth.png",
+        "born": null
+      },
+      {
+        "id": 18,
+        "firstName": "Stannis",
+        "lastName": "Baratheon",
+        "fullName": "Stannis Baratheon",
+        "title": "Lord of Dragonstone",
+        "family": "House Baratheon",
+        "imageUrl": "https://thronesapi.com/assets/images/stannis.jpg",
+        "born": "265 AL",
```

```suggestion
        "born": "265 AL"
```

A diferencia de los objetos "normales" en JS, en formato JSON las trilling commas se consideran error y por ende no sería un JSON válido!